base_layer/core/src/consensus/consensus_constants.rs

```diff
@@ -109,8 +110,6 @@ pub struct ConsensusConstants {
     permitted_output_types: &'static [OutputType],
     /// The allowlist of range proof types
     permitted_range_proof_types: [(OutputType, &'static [RangeProofType]); 5],
-    /// Coinbase outputs are allowed to have metadata, but it has the following length limit
-    coinbase_output_features_extra_max_length: u32,
```

Comment:
Why did you remove this?

Reply:
`OutputFeatures::coinbase_extra` is now typed as
```rust
pub type CoinBaseExtra = MaxSizeBytes<64>;
...rust
pub struct OutputFeatures {
...
    pub coinbase_extra: CoinBaseExtra
}
```

Reply:
Reverting